extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyDotNames.java

```diff
@@ -80,28 +80,19 @@ public boolean test(DotName dotName) {
 
     // Types ignored for reflection used by the RESTEasy and SmallRye REST client extensions.
     private static final Set<DotName> TYPES_IGNORED_FOR_REFLECTION = new HashSet<>(Arrays.asList(
-            // Jackson
-            DotName.createSimple("com.fasterxml.jackson.databind.JsonNode"),
+    // Consider adding packages below instead if it makes more sense
+    ));
 
+    private static final String[] PACKAGES_IGNORED_FOR_REFLECTION = {
```

Comment:
Wondering if we could provide a config item for user to build this list up by themselves? Of course we can have good defaults, but we have been adding classes/ packages on a continual basis here.

Reply:
Yeah, so I kinda agree. What makes me hesitant is that I fear they won't report these things anymore, thus we would have a bad experience for new users.

Reply:
Yes, but new users will ran into these warnings and report them anyway. We could either add the configuration key to the warning message to make the experience even better or once the issue is reported see if we need to review the "default values" or something more refined need to be done.